Mlem/App/Enums/PersonFlair.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  PersonFlair.swift
+//  Mlem
+//
+//  Created by Sjmarf on 28/06/2024.
+//
+
+import SwiftUI
+
+enum PersonFlair {
+    case admin
+    case bot
+    case bannedFromInstance
+    case bannedFromCommunity
+    case moderator
+    case developer
+    case op
+    
+    var color: Color {
+        switch self {
+        case .admin:
+            return .teal
+        case .moderator:
+            return Palette.main.moderation
+        case .op:
+            return .orange
+        case .bot:
+            return .indigo
```

Comment:
*nit*

Palette; useful place for accent + semantic coloring discussed in Inbox

Reply:
Oops, `PersonFlair` isn't meant to be in this PR. I'll remove it for now, it can be added in later. I agree that's a good place for the accent colors 👍